test/data/brave_ads/creative_search_result_ad.html

```diff
@@ -7,7 +7,7 @@
       "creatives": [
         {
           "@type": "SearchResultAd",
-          "data-placement-id": "data-placement-id-1",
+          "data-placement-id": "824657d0-eaed-4b80-8a42-a18c12f2977d",
```

Comment:
We have defined constants in `creative_search_result_ad_unittest_constants.h`, would be great to use the same uuid's.

Reply:
Ignore this comment, because you have many ad placements :-)